### ISSUE TITLE
Do you support DSTU2 or STU3?

### ISSUE THREAD

**Issue by sohrabhejazi** [gt-health/GT-FHIR2#68]
Hi,

You have written in the README that the program supports just STU3 and STU2 is in progress. On the other hand in the following link, you mapped OMOP CDM v5 to FHIR DSTU2:
https://github.com/gt-health/GT-FHIR2/blob/master/docs/fhir_omop_mapping.md

It is not clear to me, your program converts OMOP to  DSTU2 or STU3. Could you please explain it in more detail?

LG, Sohrab

**Comment by myungchoi** [comment]
GT-FHIR2 supports only STU3 now. GT-FHIR (which is older implementation) supports DSTU2. GT-FHIR however has some limitations because we attempted to map one-to-one mapping (FHIR resource to OMOP table mapping). In GT-FHIR2, we took totally different approach. 

Currently, we are short in staff and funding to add DSTU2 in GT-FHIR2. As soon as we find some funding, we will add DSTU2 and make more Arganaut friendly. As of now, we only support STU3. Hope this answers your question.
